merfi/backends/rpm_sign.py

```diff
@@ -57,18 +60,19 @@ def sign(self):
             raise RuntimeError('specify a --key for signing')
         logger.info('Starting path collection, looking for files to sign')
         repos = RepoCollector(self.path)
```

Comment:
Sure, we could break that up. RepoCollector is getting kind of big.

I do like the usability aspect for having `merfi` auto-discover repositories without having to explicitly specify arguments like `--rpm` or `--deb`.